apps/src/templates/progress/progressTypes.js

```diff
@@ -8,3 +8,10 @@ export const levelType = (
     icon: PropTypes.string
   })
 );
+
+export const lessonType = (
```

Comment:
This would be a good place for that JSDoc `@typedef` if you wanted to reference this type in doc annotations.